File: comms/push-notifications/device-registrations/fetch-device-registration/fetch-device-registration.php
<?php

namespace Example;

use Twilio\TwilioComms;

$client = new TwilioComms(
    accountId: '<username>',
    authToken: '<password>',
);
$client->pushNotifications->deviceRegistrations->fetch(
    'comms_device_registration_01h9krwprkeee8fzqspvwy6nq8',
);

File: comms/push-notifications/device-registrations/fetch-device-registration/fetch-device-registration.rb
require "twiliocomms"

client = TwilioComms::Client.new(
  account_id: "TWILIO_ACCOUNT_SID",
  auth_token: "TWILIO_AUTH_TOKEN"
)

client.push_notifications.device_registrations.fetch(device_registration_id: "comms_device_registration_01h9krwprkeee8fzqspvwy6nq8")
